### PR TITLE
Minor styling fixes

### PR DESCRIPTION
**Pull request by mvkv**

### CODE REVIEW

File: static/js/settings.js [hunk 1]
```diff
@@ -142,8 +142,9 @@ const updateStatusBar = (res) => {
   const HIDE_STATUS_AFTER_MS = 2000;
   const statusBar = document.getElementById("status-bar");
   const statusText =
-    res.status === 200 ? "Success!" : "Failure, please try again!";
+    res.status === 200 ? "Success!" : `Failure, ${res.error || 'Unknown error'}`;
```
Comment (powahftw) [inline static/js/settings.js]: I'd use null coalescing operator ??

File: static/js/settings.js [hunk 2]
```diff
@@ -153,6 +154,10 @@ const updateStatusBar = (res) => {
 };
 
 const postUpdatedSettings = async (payload) => {
+  if (!payload.session_id) {
```
Comment (powahftw) [inline static/js/settings.js]: 'session_id' in payload is better.

**Review by powahftw — CHANGES_REQUESTED** [review]

File: static/js/settings.js (hunk 1 above)
Comment (mvkv) [reply static/js/settings.js]: Done

File: static/js/settings.js (hunk 2 above)
Comment (mvkv) [reply static/js/settings.js]: Done

**Review by powahftw — APPROVED** [review]

**Pull request merged**